tests/sources/fixtures/notion/requirements.txt

```diff
@@ -0,0 +1 @@
+flask==2.3.2
```

Comment:
I don't think this is necessary, since this is in `requirements/ftest.txt`, right?